cmd/gravy/simulate.go

```diff
@@ -2,20 +2,38 @@ package main
 
 import (
 	"bufio"
-	"flag"
 	"log"
 	"strconv"
 
 	"github.com/d-sparks/gravy/algorithm"
 	"github.com/d-sparks/gravy/db"
 	"github.com/d-sparks/gravy/db/dailywindow"
-	"github.com/d-sparks/gravy/exchange"
 	"github.com/d-sparks/gravy/gravyutil"
+	"github.com/d-sparks/gravy/mock"
+	"github.com/spf13/cobra"
 )
 
-var windows = flag.String("windows", "./data/kaggle/historical_as_windows.json", "Kaggledata")
-var symbols = flag.String("symbols", "./data/kaggle/historical_stocks.csv", "Stock symbols")
-var output = flag.String("output", "./results", "Results output")
+var simulateCmd = &cobra.Command{
```

Comment:
Did you check that this still works? `go run cmd/simulate/simulate.go` makes a ./results file with a reasonable CSV of dates and values?

(I'll start adding tests for code going forward and whenever we get to it we can add tests for old stuff as well.)

Reply:
Actually, I'm going to resolve this. Simulate probably doesn't work because I probably left it in a halfway state. We can merge this pull request and I'll get simulate to work afterward (I have a bunch of changes in my branch as well).